src/background/extension-main.js

```diff
@@ -134,15 +134,7 @@ class BackgroundApp {
     }
     static _loadConfiguration() {
```

Comment:
I think you can go ahead and completely remove this `_loadConfiguration()` function? It is probably only used from within this file.

Reply:
Good point! removed